Uplift/Controllers/HomeViewController.swift

```diff
@@ -69,23 +74,42 @@ class HomeViewController: UIViewController {
 
             // Reload All Gyms section
             self.collectionView.reloadSections(IndexSet(integer: 0))
+            
+            self.pendingNetworkRequests -= 1
+            if self.pendingNetworkRequests == 0 {
+                self.loadingHeader.isHidden = true
+                self.loadingScrollView.isHidden = true
+            }
         }
 
         // Get Today's Classes
         let stringDate = Date.getNowString()
-
+        pendingNetworkRequests += 1
         NetworkManager.shared.getGymClassesForDate(date: stringDate, completion: { (gymClassInstances) in
             self.gymClassInstances = gymClassInstances.sorted { (first, second) in
                 return first.startTime < second.startTime
             }
 
             // Reload Today's Classes section
             self.collectionView.reloadSections(IndexSet(integer: 1))
+            
+            self.pendingNetworkRequests -= 1
+            if self.pendingNetworkRequests == 0 {
+                self.loadingHeader.isHidden = true
+                self.loadingScrollView.isHidden = true
+            }
```

Comment:
since we use this same code multiple times, can we extract out to a function called `decrementNumPendingNetworkRequests()`